### ISSUE TITLE
swagger UI3 (served by nswag) does not work for us.... IE11/Edge on Win10

### ISSUE THREAD

**Issue by ddewaleffe** [RSuter/NSwag#1727]
Hello, we started to use nswag quite recently... 
"Generated using the NSwag toolchain v11.20.0.0 (NJsonSchema v9.11.0.0 (Newtonsoft.Json v9.0.0.0))"
the app expose swagger-ui3 3 but it only partially works...

We have a number of spots in the UI where we have a message like [Could not render this component, see the console.]

Looking at the console we see messages like : description "Object doesn't support property or method 'map'", or [with all details]...

[object Error]: {description: "Object doesn't support property or method 'get'", message: "Object doesn't support property or method 'get'", number: -2146827850, stack: "TypeError: Object doesn't support property or method 'get' at value (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:76:57750) at t.prototype.render (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:61158) at w._renderValidatedComponentWithoutOwnerOrContext (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:427717) at w._renderValidatedComponent (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:427826) at w.performInitialMount (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:423569) at w.mountComponent (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:422466) at i.mountComponent (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:13:34359) at c.Mixin.mountChildren (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:418096) at X.Mixin._createInitialChildren (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:407391) at X.Mixin.mountComponent (https://localhost:44360/swagger/facade/swagger-ui-bundle.js:70:405619)"}

For searching a little, it would seem extra stuff [es6-shim js stuff] would be missing from what is being served.....  See: [Stack Overflow](https://stackoverflow.com/questions/35429768/object-doesnt-support-property-or-method-keys-ie11)

Basically, spots where we should be able to add input values are not rendered. making the thing not usable...

Is this a known issue? 

Any hint/help/solution welcome...

BTW, we cannot use another browser....

Thanks,

D.

**Comment by RSuter** [comment]
IE11 and Edge are two completely different browsers, … you should move this issue to the Swagger UI 3 repo; NSwag just embeds Swagger UI 3...

**Comment by RSuter** [comment]
Maybe we can just add the shims to index.html?

**Comment by ddewaleffe** [comment]
Maybe... Do you see an easy way for us to try this?

**Comment by RSuter** [comment]
- Clone the repo
- open NSwag.Min.sln
- Modify index.html in NSwag.AspNetCore
- Start and test Sample.NetCore21

**Comment by ddewaleffe** [comment]
Will try this over the w-e on my machine and will verify at the office
during next week.
Will follow-up as soon as I have results.
Thanks.
-----------------------------------------
Dominique de Waleffe
GSM:+32 477 533 521
dominique@4d-consult.com

**Comment by ddewaleffe** [comment]
Took me a while to get started.... Had to find out global.json needed
changes...
I confirm that the UI3 work on Edge and IE 11 versions on my machine [win
10 prof, latest updates].
- The components are rendered as they should.

So maybe what we are seeing is a consequence of something else in the
office environment [win 10 version, IE/Edge GPO's, network restrictions, AV
products].
Will dig some more at work.

Meanwhile I think you can close the issue.
If/when I find something useful about the reason for failure, I'll report
as extra comment...

Thanks for your help.

**Comment by RSuter** [comment]
I recently updated swagger ui 3 in master branch and i think this is not yet relaesed...

**Comment by ddewaleffe** [comment]
Ah!  I see you changed this on sept 20. And if I understand well, was
released on sept 29 in v11.20.0.0 which is the version I used.
